spec.md

```diff
@@ -615,6 +621,24 @@ message PublishVolumeInfo {
   // possible.
   map<string, string> values = 1;
 }
+
+message Principal {
+  // describes attributes for defining the principal identity of
+  // a volume binding for a container.
+
+  // This field is OPTIONAL. If specified it contains the CO namespace
+  // identity in which the the following identity information (user /
+  // group) exists.
+  string namespace = 1;
+  // This field is OPTIONAL. If specified it contains the CO user
+  // identity holding ownership of a volume, similar to the user_id
```

Comment:
A `string user` implies that this is a textual label, like `"james"`. But when I see `user_id` in the commentary for this field, I think of a numerical ID like `600`.

Is it the intent that this field should only carry textual labels, or numerical IDs, or both (and the plugin just has to deal with whatever the CO spits out)?

We also clearly call out in the spec that we're not aiming for POSIX compliance. Is there a need to mention POSIX at all in these comments?

(ditto for `group` and its comments)